tools/pd-ctl/pdctl/command/region_command.go

```diff
@@ -414,7 +414,7 @@ func showRegionsFromStartKeyCommandFunc(cmd *cobra.Command, args []string) {
 // NewRegionWithCheckCommand returns a region with check subcommand of regionCmd
 func NewRegionWithCheckCommand() *cobra.Command {
 	r := &cobra.Command{
-		Use:   "check [miss-peer|extra-peer|down-peer|pending-peer|incorrect-ns]",
+		Use:   "check [miss-peer|extra-peer|down-peer|pending-peer|incorrect-ns|offline-peer|empty-region]",
```

Comment:
Oh, I also find that the `incorrect-ns` is no longer needed, would you mind removing it and updating the API document? You can follow the steps listed [here](https://github.com/pingcap/pd/blob/master/docs/development.md#updating-api-documentation).

Reply:
@rleungx PTAL.thanks